kiosk/src/Components/ScanQR.tsx

```diff
@@ -50,26 +70,29 @@ const ScanQR: React.FC<IProps> = ({ kiosk }) => {
 
     return (
         <div className="scanQrPage">
-            <h2>Add game to </h2>
-            <h1>Kiosk {kioskId}</h1>
+            <h2>Add game to<br/>Kiosk {kioskId}</h2>
             <div className="scanInstructions">
-                <div className="qrOption">
-                    <p className="scanIntro">Scan game's share QR code with the button below</p>
-                    {
-                        !scannerVisible &&
-                        <button className="scanQrButton" onClick={renderQrScanner} >Scan QR code</button>
-                    }
-                    <div id="qrReader"></div>
-                    {
-                        scannerVisible &&
-                        <p className="scanTip">Tip: Do not use the kiosk's QR code</p>
-                    }
-                </div>
-
                 {
-                    !phoneWidth &&
+                    !linkVisible &&
+                    <div className="qrOption">
+                        <p className="scanIntro">Scan game's share QR code with the button below</p>
```

Comment:
Nit: should this text change once the user has clicked the button? Maybe to just "Scan game's share QR code".

Reply:
Hmm. I have a hard time believing that users would be confused by the text still being there, but I can also see that it is redundant, with the button saying to scan the qr code. I think based on the options, that keeping the text be as is just clutters the page. 

Maybe something like "Go to share your game in MakeCode arcade to:". I'll have to chat with Claire about possibilities here.

Reply:
We decided to just get rid of that text completely. In the flow for the user, they should understand what to do on this page without that text. Thanks for the call out, Kim! I think this will be a much nicer user experience 😄.

Reply:
Oh really? I think there should be some text. Imagine it being the first time for a user. They might click that button and then have no reference for what to scan or where to look for the QR code. Unless that help link is still visible? Then that might be enough context.

Reply:
I guess they could also hit 'cancel' and then see the previous text again.

Reply:
Yeah, I was a bit worried about that, as well, but the link should be able to help. It is also important to note that this page is very hard to get to outside of context of the kiosk. I think the content of the header and the link should be enough to point the user in the right direction. There are also the directions on the add game page on the kiosk that can provide more context.

This might be something we want to test, though...

Reply:
I'm going to go forward with the no-text design for now, but I can get some user feedback and see what comes of it.